app/views/components/table_rulings.scala.html

```diff
@@ -58,7 +58,7 @@
                 <tbody id="rulings-list-body">
                     @for((c: Case, index: Int) <- cases.results.zipWithIndex) {
                         <tr id="rulings-list-row-@{index}">
-                            <td id="rulings-list-row-@{index}-reference">@c.reference</td>
+                            <td id="rulings-list-row-@{index}-reference">GB@c.reference</td>
```

Comment:
We need to double check with Joe if GB should be part of the reference. We have removed it from lots of places in the past but I know Steve P still has it in his prototypes

Reply:
After post-amigos this has been removed now, so no longer a thing